test/build/yarn-workspaces/yarn.lock

```diff
@@ -0,0 +1,8 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
all of our other tests only use npm, lets convert to package-lock

Reply:
Wouldn’t it make sense to use `yarn` in this one instance since it’s testing a Yarn feature?

Reply:
🤦 that does make sense, ignore me!